### PR TITLE
added front-end and back-end of web component app

### PR DESCRIPTION
**Pull request by sumitsaurabh927**
Hey @unicodeveloper please take a look!

### CODE REVIEW

**Review by unicodeveloper — COMMENTED** [review]
Please rename the folder to "web-component-notification-center-example"

**Comment by sumitsaurabh927** [comment]
done in this commit: 9f4fa0f9177d8026ca6a2d0cec8fa755a946afd2

**Comment by unicodeveloper** [comment]
Please add a readme on how to clone and run the app. You can [follow this example](https://github.com/novuhq/examples/tree/main/vue-nextjs-trigger-inapp-notifications#readme)

**Comment by sumitsaurabh927** [comment]
@unicodeveloper done! ✅

File: web-component-notification-center-example/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+You can read more about it [here](https://docs.novu.co/notification-center/web-component/)
+
+## How to run this app:
+
+Running this app on your local machine is quite simple. You just need to go through the following steps:
+
+- Clone this repo to your machine (make sure you've clone both the front-end as well as the back-end to your machine)
+- run `npm install` in the root of backend to install all the packages
+- Now, you go over to [Novu](www.novu.co) to get your API key and add it to the `.env` file in the root of the front-end repo under this variable name `NOVU_API_KEY`
```
Comment (unicodeveloper) [inline web-component-notification-center-example/README.md:14]: Replace the url here with web.novu.co

File: web-component-notification-center-example/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+This app uses Novu's web component to add a notification center to any web app. The best part about it is that it can be used in any web app, no matter what tech stack you've used.
+
+You can read more about it [here](https://docs.novu.co/notification-center/web-component/)
+
+## How to run this app:
+
+Running this app on your local machine is quite simple. You just need to go through the following steps:
+
+- Clone this repo to your machine (make sure you've clone both the front-end as well as the back-end to your machine)
```
Comment (unicodeveloper) [inline web-component-notification-center-example/README.md:12]: => make sure you've cloned both the front-end

File: web-component-notification-center-example/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- run `npm install` in the root of backend to install all the packages
+- Now, you go over to [Novu](www.novu.co) to get your API key and add it to the `.env` file in the root of the front-end repo under this variable name `NOVU_API_KEY`
+- Repeat the last step for the backend repo as well with the same variable name.
```
Comment (unicodeveloper) [inline web-component-notification-center-example/README.md:15]: Should this not be frontend?

File: web-component-notification-center-example/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+- Create a database on MongoDB.
+- Also add your __MongoDB connection URL__ to the backend's _.env_ file under the variable name `CONNECTION_URL`. You can get it from the MongoDB database you've created in a previous step.
+- Now test the following end-point on Postman: `https://notificationsgeneratorbackend.onrender.com/home` and don't forget to add `{"description":"testing"}` in the request. 
+
+You can also go throught a detailed article that also covers deploying the frontend as well as the backend [here](https://dev.to/novu/how-to-add-in-app-notifications-to-any-web-app-1b4n)
```
Comment (unicodeveloper) [inline web-component-notification-center-example/README.md:20]: => through

**Review by unicodeveloper — APPROVED** [review]

**Pull request merged**